### DISCUSSION TITLE
Are there any anti-patterns that I shouldn't use in sveltekit?

### DISCUSSION THREAD

**Discussion by kimud6003** [sveltejs/kit#10372] [Q&A]
For example, I would like to know about some inconvenient precautions such as making and using DOM by myself, or managing all states as a store.

**Comment by jtlapp** [comment]
One of my first mistakes with Svelte was placing generic modals in the layout and having different pages pop up that modal with page-specific information. These modals would sometimes survive page changes, and managing state became tricky. I eventually learned to embed the modal on/off logic separately in each page that having a modal.
